appserver/deployment/dol/pom.xml

```diff
@@ -141,6 +141,18 @@
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
             <version>2.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>javax.inject</groupId>
+                    <artifactId>javax.inject</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <!-- Replace javax.inject dependency from cdi api -->
```

Comment:
otherwise classpath in MANIFEST.MF is generated with javax.inject.jar, but this jar is no longer in GF

Reply:
jakarta.inject.jar is there

Reply:
Is the bad classpath entry in the manifest causing any issue ?
Note that instead of excluding this for every occurence of cdi-api, we should rather manage the dependency under appserver/pom.xml and do the exclusion once.

Reply:
Yes, javax.inject package can not be loaded when bad classpath.

Reply:
How ? This is OSGi the Class-Path should not have an impact.

Reply:
it has

Reply:
Can you share a stack trace ?

Reply:
WARNING: Can not start bundle file:/X:/Projects/Jersey/test/orig/glassfish5/glassfish/modules/javax.inject.jar because it is not contained in the list of installed bundles.

Reply:
This might have been some other cause. Anyway, you fight here to have a wrong classpath?

Reply:
No, I fight to fix the right problems the right way :)
Clearly the error above highlight some issue with the OSGi configuration.

Don't try to fix all GlassFish poms as part of a jersey integration.

Reply:
Split #22678